src/util/DOMUtils.ts

```diff
@@ -73,14 +73,23 @@ export const getStyleString = (style: CSSProperties) =>
     '',
   );
 
+function removeInvalidKeys(obj: Record<string, any>) {
+  const copyObj = { ...obj };
+  Object.keys(copyObj).forEach(key => {
+    if (!copyObj[key]) {
```

Comment:
I'm confused, why would this ever not be true if you just called object.keys? Only if someone passed in. A falsy value?

Reply:
In functions like getTicks, we are passing undefined or an empty string to font-size.

Reply:
Then I wonder, should we even call getStringSize in those cases?

Reply:
It's an issue stemming from not knowing the calculated font size before the DOM is rendered. If the component waits for rendering in the DOM and calculates it all at once, the problem should be resolved. With the ongoing refactorings, it might naturally get resolved, don't you think?